server.js

```diff
@@ -3092,6 +3095,55 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// GitHub release date integration.
+camp.route(/^\/github\/release-date\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var user = match[1];  // eg, qubyte/rubidium
+  var repo = match[2];
+  var format = match[3];
+  var apiUrl = githubApiUrl + '/repos/' + user + '/' + repo + '/releases/latest';
+  var badgeData = getBadgeData('release date', data);
+  if (badgeData.template === 'social') {
+    badgeData.logo = badgeData.logo || logos.github;
+  }
+  githubAuth.request(request, apiUrl, {}, function(err, res, buffer) {
+    if (err != null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(format, badgeData);
+      return;
+    }
+    try {
+      var data = JSON.parse(buffer);
+
+      if (typeof data.created_at !== 'string') throw 'Project has no releases';
```

Comment:
Rather than `throw` to land in the `catch` block, it would be more direct and clearer to invoke `sendBadge` and return early, as above.